gnuefi/crt0-efi-ia32-local.S

```diff
@@ -163,11 +163,12 @@ _start:
 // hand-craft a dummy .reloc section so EFI knows it's a relocatable executable:
  
  	.data
-dummy:	.4byte	0
+dummy0:	.4byte	0
+dummy1:	.4byte	0
 
 #define IMAGE_REL_ABSOLUTE	0
- 	.section .reloc
- 	.4byte	dummy					// Page RVA
+ 	.section .areloc, "a", %progbits
+ 	.4byte	dummy1 - dummy0					// Page RVA
```

Comment:
This looks wrong it's supposed to be Page RVA even if it gets ignored anyways
Should be dummy0??

Reply:
Likely wrong but assembly symbols don't calculate